<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParcelsFromPolygonDialogBase</class>
 <widget class="QDialog" name="ParcelsFromPolygonDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parcels from polygon</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/ParcelsFromPolygon/icon.png</normaloff>:/plugins/ParcelsFromPolygon/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Direction</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>First Direction</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_Radians">
            <property name="text">
             <string>Radians</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_Degrees">
            <property name="text">
             <string>Degrees</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_Gradians">
            <property name="text">
             <string>Gradians</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_PickFirstDir">
              <property name="text">
               <string>Pick</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="counter_Radians">
            <property name="decimals">
             <number>16</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="counter_Degrees">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>16</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="counter_Gradians">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>16</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Then</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_PostKeepConstant">
            <property name="text">
             <string>keep it constant</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_PostConst">
            <property name="text">
             <string>advance by </string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_PostProportional">
            <property name="text">
             <string>proportional to area until</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_PickLastDirection">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Pick</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="counter_PostConst">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>16</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="counter_PostProportional">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>16</number>
            </property>
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Area</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Each line should contain the area for one parcel</string>
         </property>
         <property name="buddy">
          <cstring>plainTextEdit_Areas</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_Areas">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_ParcelForRest">
         <property name="text">
          <string>Create a parcel for the rest of the polygon</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Polygon</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Parcel selected polygon in current layer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioButton_Radians</tabstop>
  <tabstop>counter_Radians</tabstop>
  <tabstop>radioButton_Degrees</tabstop>
  <tabstop>counter_Degrees</tabstop>
  <tabstop>radioButton_Gradians</tabstop>
  <tabstop>counter_Gradians</tabstop>
  <tabstop>pushButton_PickFirstDir</tabstop>
  <tabstop>radioButton_PostKeepConstant</tabstop>
  <tabstop>radioButton_PostConst</tabstop>
  <tabstop>counter_PostConst</tabstop>
  <tabstop>radioButton_PostProportional</tabstop>
  <tabstop>counter_PostProportional</tabstop>
  <tabstop>pushButton_PickLastDirection</tabstop>
  <tabstop>button_box</tabstop>
  <tabstop>plainTextEdit_Areas</tabstop>
  <tabstop>checkBox_ParcelForRest</tabstop>
 </tabstops>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ParcelsFromPolygonDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ParcelsFromPolygonDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Radians</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Radians</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Radians</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Degrees</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Radians</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Gradians</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Degrees</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Radians</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Degrees</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Degrees</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Degrees</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Gradians</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Gradians</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Radians</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Gradians</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Degrees</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_Gradians</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_Gradians</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostKeepConstant</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_PostConst</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostKeepConstant</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_PostProportional</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostConst</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_PostConst</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostConst</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_PostProportional</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostProportional</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_PostConst</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostProportional</sender>
   <signal>toggled(bool)</signal>
   <receiver>counter_PostProportional</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostKeepConstant</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_PickLastDirection</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostConst</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_PickLastDirection</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_PostProportional</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_PickLastDirection</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
